linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/oval/shared.xml

```diff
@@ -49,7 +49,7 @@
   <ind:textfilecontent54_object id="obj_sshd_clientalivecountmax" version="2">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
     <ind:pattern operation="pattern match">^[\s]*(?i)ClientAliveCountMax[\s]+([\d]+)[\s]*(?:#.*)?$</ind:pattern>
-    <ind:instance datatype="int">1</ind:instance>
+    <ind:instance operation="greater than or equal" datatype="int">1</ind:instance>
```

Comment:
The text of the rule states:

> If "ClientAliveCountMax" do [sic] not exist, is not set to a value of "1" in "/etc/ssh/sshd_config", or is commented out, this is a finding.

The `operation="greater than or equal"` seems to contradict that.